### PR TITLE
📝 Update swagger file

### PR DESCRIPTION
**Pull request by hirsch88**
- Add api info endpoint
- Adjust method usage
- Add admin endpoint to alter the games
- Add new security concept

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -127,15 +149,35 @@ paths:
           description: "Successful operation"
         400:
           description: "Invalid password credentials"
+  /auth/verify/{verifyToken}:
+    put:
+      tags:
+      - "auth"
+      summary: "Verifys the user throw the email address"
```
Comment (keniseli) [inline swagger.yml]: "Verifies the user through the email address" ;-)

File: swagger.yml [hunk 2]
```diff
@@ -148,7 +190,7 @@ paths:
   /users:
     get:
       security:
-        - basicAuth: []
+        - bearerAuth: []
       tags:
       - "users"
       summary: "Retunrs a list of users"
```
Comment (keniseli) [inline swagger.yml:196]: Typo

**Review by keniseli — APPROVED** [review]
Made some changes

File: swagger.yml (hunk 2 above)
Comment (hirsch88) [reply swagger.yml:196]: No. Only the login endpoint has basic auth.

**Pull request merged**